packages/nimble-components/src/rich-text-mention/mention-users/tests/rich-text-mention-users.stories.ts

```diff
@@ -0,0 +1,30 @@
+import { html } from '@microsoft/fast-element';
+import type { Meta, StoryObj } from '@storybook/html';
+import { createUserSelectedThemeStory } from '../../../utilities/tests/storybook';
+import { hiddenWrapper } from '../../../utilities/tests/hidden';
+
+const metadata: Meta = {
+    title: 'Internal/Rich Text',
```

Comment:
It's best to keep the titles in sync with the file names
```suggestion
    title: 'Internal/Rich Text Mention Users',
```

Reply:
Updated